cloud-functions/http_standardize_names/main.py

```diff
@@ -14,36 +18,18 @@ def http_standardize_names(request: flask.Request) -> flask.Response:
     :param data: Either a FHIR bundle or a FHIR-formatted JSON dict
     :return: The bundle or resource with names appropriately standardized
     """
+    content_type = "application/json"
     # Validate request header.
-    if request.headers.get("Content-Type") != "application/json":
-        logging.error("Header must inclue: 'Content-Type:application/json'.")
-        return {
-            "status": 400,
-            "summary": "Bad request",
-            "description": "Header must inclue: 'Content-Type:application/json'.",
-        }
-
-    try:
-        request_json = request.get_json(silent=False)
-    except AttributeError as error:
-        logging.error(error)
-
-        return {
-            "status": 400,
-            "summary": "Invalid request body",
-            "description": "Invalid JSON",
-        }
+    header_response = validate_request_header(request, content_type)
 
     # Check that the request body contains a FHIR bundle or resource.
-    if request_json.get("resourceType") is None:
-        error_message = (
-            "FHIR Resource Type not specified. "
-            + "The request body must contain a valid FHIR bundle or resource."
-        )
-        logging.error(error_message)
-        return {"status": 400, "summary": "Bad request", "description": error_message}
+    if header_response.status_code == 400:
+        return header_response
 
-    # Perform the name standardization
-    response = standardize_names(request_json)
+    body_response = validate_fhir_bundle_or_resource(request)
+    if body_response.status_code != 400:
+        request_json = request.get_json(silent=False)
+        # Perform the name standardization
+        body_response = _full_response(standardize_names(request_json))
```

Comment:
nit: I believe a leading underscore indicates an internal method in Python. Importing this outside of the utils module seems to suggest that it isn't just an internal method. Should we consider renaming? Also, my preference would be for the function name to be a little more descriptive to make the purpose clearer to users. Maybe something like `make_json_content_response`? I'm just not quite sure that `_full_response` makes it clear what this function does.

Reply:
I have fixed this in another PR specific to the utils package.  I was thinking about this last night and my plan was to get rid of full_response all together and just use either success or fail as responses.  I think this is a cleaner method.  Thoughts?

Reply:
Yep, that works for me.